src/config.ts

```diff
@@ -256,4 +258,4 @@ export interface TaggedList {
   [abbrev: string]: string;
 }
 
-export let config = new JuvixConfig();
+export const config = new JuvixConfig();
```

Comment:
Can you test yourself installing Juvix with the extension (in a fresh environment) after this change? I'm concerned if this change breaks that.